README.markdown

```diff
@@ -8,7 +8,7 @@ If you're a computer science student who needs to learn this stuff for exams --
 
 The goal of this project is to **explain how algorithms work**. The focus is on clarity and readability of the code, not on making a reusable library that you can drop into your own projects. That said, most of the code should be ready for production use but you may need to tweak it to fit into your own codebase.
 
-Most code is compatible with **Xcode 8.3** and **Swift 3**. We'll keep this updated with the latest version of Swift.
+All code are compatible with **Xcode 9** and **Swift 4**. We'll keep this updated with the latest version of Swift.
```

Comment:
The word "code" is used as a [mass noun](https://en.wikipedia.org/wiki/Mass_noun). You should consider it as a singular quantity.